noxfiles/dev_nox.py

```diff
@@ -0,0 +1,42 @@
+"""Contains the nox sessions for running development environments."""
+import nox
+from constants_nox import COMPOSE_SERVICE_NAME, RUN
+from docker_nox import build
+from run_infrastructure import run_infrastructure
+
+
```

Comment:
I may be missing it, but I don't see the equivalent of `make server`. I see people using that one regularly so it would be worth adding.

Reply:
`make server` is just `docker compose up`, in my opinion that's not really worth having an alias for

Reply:
the dev commands are there to simplify some potentially longer commands/tasks, but I still think the team should be comfortable calling out to docker/docker compose directly when it makes sense